src/cljs/replumb/repl.cljs

```diff
@@ -570,6 +570,22 @@
       (call-back (common/wrap-success (s/join \newline vars)))
       (call-back (common/wrap-success "nil")))))
 
+(defn process-apropos
+  [opts cb data str-or-pattern]
+  (let [matches? (if (instance? js/RegExp str-or-pattern)
+                   #(re-find str-or-pattern (str %))
+                   #(< -1 (.indexOf (str %) (str str-or-pattern))))
+        defs (->> (known-namespaces)
+                  (mapcat (fn [ns]
+                            (let [ns-name (str ns)]
+                              (map #(symbol ns-name (str %))
+                                   (filter matches? (keys (ns-publics st ns)))))))
+                  sort)
+        call-back (partial call-back! (merge opts {:no-pr-str-on-value true}) cb data)]
+    (if (seq defs)
+      (call-back (common/wrap-success (str defs)))
+      (call-back (common/wrap-success "nil")))))
```

Comment:
I guess this `if` is not necessary, it look like you can call `(common/wrap-success (seq defs))`. But try because it might be completely wrong on my side.